portfolio/src/main/webapp/style.css

```diff
@@ -32,13 +31,11 @@ body {
     width: 100%;
 }
 
-
 /* navigation bar */
 ul {
     list-style-type: none;
     margin: 0;
     padding: 0;
-
 }
 
 .navBar {
```

Comment:
2 CSS nits:
- Use hypens to separate words in ID and class names (https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters)
- If possible, avoid unnecessarily qualifying ID and class names (https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors)